apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/SimpleManagedTracer.cs

```diff
@@ -145,6 +145,23 @@ public string GetCurrentTraceId()
             return _trace.TraceId;
         }
 
+        /// <inheritdoc />
+        public ulong? GetCurrentSpanId()
+        {
+            lock (_stackMutex)
```

Comment:
I changed the name to _stackLock.  We have a tracer per incoming request so I don't *think* this should cause huge issues but I am happy to revisit or change it if you feel otherwise.